packages/gestalt/src/TextField.css

```diff
@@ -9,17 +9,26 @@
   color: var(--g-colorGray200);
 }
 
-.unstyledTextField {
+.unstyledTextField, .measure {
   composes: unstyled from "./FormElement.css";
   composes: Text fontSize3 from "./Text.css";
-  flex: 1;
   margin-bottom: 4px;
 }
 
+.unstyledTextField {
+  composes: absolute left0 top0 right0 bottom0 from "./Layout.css";
+  width: 100%;
+}
+
 .unstyledTextField:focus {
   outline: 0;
 }
 
+.measure {
```

Comment:
Only thing I'm not totally loving is this name "measure", as I don't think it does a great job explaining the intent behind this invisible div. Maybe `textFieldSpacer` or `textFieldWrapper` or something similar will help future devs better understand the purpose